Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -165,11 +165,16 @@ public sealed class NetworkIdentity : MonoBehaviour
         /// </summary>
         public bool hasAuthority { get; internal set; }
 
-        /// <summary>
-        /// The set of network connections (players) that can see this object.
-        /// <para>null until OnStartServer was called. this is necessary for SendTo* to work properly in server-only mode.</para>
-        /// </summary>
-        public Dictionary<int, NetworkConnection> observers;
+        // interest management
+        internal HashSet<NetworkConnectionToClient> rebuild = new HashSet<NetworkConnectionToClient>();
+        public HashSet<NetworkConnectionToClient> observersx = new HashSet<NetworkConnectionToClient>();
+
+        // Enable to temporarily hide from players. For example, to hide a
+        // monster while it's respawning. This way the monster still exists,
+        // still gets updated, but is hidden from players. This is more
+        // convenient than removing the monster completely and respawning it
+        // with a Spawner.
+        [HideInInspector] public bool forceHidden;
```

Comment:
Would be better to have an extra monobehaviour like "interest management settings" that can be customized the user. Some interest management wont need `forceHidden` and others may need different settings, and some objects might need `forceShown` (for example shared UI like a scoreboard)